pages/index.tsx

```diff
@@ -20,18 +20,51 @@ import { StakingContractCode } from '../helpers/runkit';
 
 ChartJS.register([CategoryScale, LinearScale, PointElement, LineElement]);
 
+interface StorageProps {
+  newValue: string | number | undefined,
+  key: string,
+  floatValue: number,
+  currentValue: number | string
+}
+
 export default function Home() {
   const [totalDevelopers, setTotalDevelopers] = useState(0);
   const [inProgress, setInProgress] = useState(0)
-  const [beproStaked, setBeproStaked] = useState(0)
-  const [onNetwork, setOnNetwork] = useState(0)
+  const [beproStaked, setBeproStaked] = useState<number | string>(0)
+  const [onNetwork, setOnNetwork] = useState<number | string>(0)
+  const [totalAmount, setTotalAmount] = useState<number | string>(0);
   const { getTotalDevelopers } = useApi()
+
+
+  function handleLocalStorage({floatValue, newValue, key, currentValue}: StorageProps) {
```

Comment:
we don't need a "floatValue" since we will never interact with it -- all we need is the represented value